src/editor/user-tasks/user-task.js

```diff
@@ -1,6 +1,6 @@
 const TASK_ERRORS = {
   noCallback:
-    "you must provide either a callback, a keybindingCalblack, or both (if the callbacks differ)",
+    "you must provide either a callback, a keybindingCallblack, or both (if the callbacks differ)",
```

Comment:
typo – `keybindingCallback`

Reply:
hah, my correction only did half the job